app/components/floating_text_input_label/index.tsx

```diff
@@ -100,7 +100,7 @@ type FloatingTextInputProps = TextInputProps & {
     error?: string;
     errorIcon?: string;
     isKeyboardInput?: boolean;
-    label: string;
```

Comment:
Not convinced by not having the label, deferred to UX

Reply:
UX pointed out that the no label is expected, I this is best to use a TextInput instead of modifying thid component